lib/multi-acc-new-eks-mixed-observability-pattern/GrafanaOperatorConstruct.ts

```diff
@@ -0,0 +1,53 @@
+import { Construct } from 'constructs';
```

Comment:
Why is this name GrafanaOperator Construct?

Reply:
renamed to grafana-operator-index.ts. This creates EKS cluster with grafana-operator and external-secrets-operator